x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -71,13 +72,40 @@ const i18nTexts = {
         'Go to your logs directory to view the deprecation logs or enable log collecting to see them in the UI.',
     }
   ),
+  deniedPrivilegeTitle: i18n.translate(
+    'xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeTitle',
+    {
+      defaultMessage: `You're missing index privileges`,
+    }
+  ),
+  deniedPrivilegeDescription: (privilegesMissing: MissingPrivileges) => (
+    // NOTE: hardcoding the missing privilege because the WithPrivileges HOC
```

Comment:
Im still a bit confused by the nomenclature, but as I understand an index can have many privileges: create, read, write, etc. So with this comment I meant to write that the implementation of the HOC doesn't allow us to specifically surface which ones of those the user doesn't have. At least without having to make a change that is perhaps out of scope for this PR since it would affect other apps also.

Reply:
Ah, I understand now. Thanks for explaining!